src/main/java/com/bankbazaar/test/controller/DepartmentController.java

```diff
@@ -0,0 +1,49 @@
+package com.bankbazaar.test.controller;
+
+import com.bankbazaar.test.entity.Department;
+import com.bankbazaar.test.service.DepartmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class DepartmentController {
+    @Autowired
+    private DepartmentService service;
+
+    @PostMapping("/addDepartment")
+    public Department addDepartment(@RequestBody Department department) {
+        return service.saveDepartment(department);
+    }
+
+    @PostMapping("/addDepartmentList")
```

Comment:
No need to have api for multiple insert together

Reply:
I have reoved this api